lib/PuppeteerSharp/IBrowserOptions.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace PuppeteerSharp
+{
+    public interface IBrowserOptions
+    {
+        /// <summary>
+        /// Whether to ignore HTTPS errors during navigation. Defaults to false.
+        /// </summary>
+        bool IgnoreHTTPSErrors { get; }
+        bool AppMode { get; }
```

Comment:
Missing XML Doc

Reply:
From what I can see, if it's false, then the viewport is set to 800x600, does it have other uses?

Reply:
That property is poorly documented in our friend repo. Then only usage is here https://github.com/kblok/puppeteer-sharp/blob/d98f9c60bfd580d7d1d5619f3ea167bae978dabe/lib/PuppeteerSharp/Launcher.cs#L69
I'd put something like `Set Headless in false when true and enable automation when false`